plugins/out_stackdriver/stackdriver.c

```diff
@@ -2410,8 +2422,13 @@ static flb_sds_t stackdriver_format(struct flb_stackdriver *ctx,
         }
 
         /* logName */
```

Comment:
This comment needs to be moved down to where the `logName` section is (line 2432)

Reply:
missed that one... Just pushed the latest commit!